### PR TITLE
Added travis integration and README

### PR DESCRIPTION
**Pull request by chriswvu**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "2.6"
+  - "2.7"
+  - "3.3"
+  - "3.4"
+sudo: false
+install: true
+script: true
```
Comment (lorengordon) [inline .travis.yml]: The red circle there is indicating that this file is missing a new line character at the end of the last line in the file.

Comment (chriswvu) [inline .travis.yml]: I see now, I'll edit the file.

Chris

On Fri, Feb 26, 2016 at 9:04 AM, Loren Gordon <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/plus3it/systemprep-content/pull/8#discussion_r54245851>
> :
>
> > @@ -0,0 +1,9 @@
> > +language: python
> > +python:
> > +  - "2.6"
> > +  - "2.7"
> > +  - "3.3"
> > +  - "3.4"
> > +sudo: false
> > +install: true
> > +script: true
>
> The red circle there is indicating that this file is missing a new line
> character at the end of the last line in the file.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/plus3it/systemprep-content/pull/8/files#r54245851>.
>



-- 
Plus3 IT Systems - Cloud Engineer
304-279-4771
christopher.stotler@plus3it.com

-- 
 

------------------------------

*This e-mail and any attachments are intended only for the use of the 
addressee(s) named herein and may contain proprietary information. If you 
are not the intended recipient of this e-mail or believe that you received 
this email in error, please take immediate action to notify the sender of 
the apparent error by reply e-mail; permanently delete the e-mail and any 
attachments from your computer; and do not disseminate, distribute, use, or 
copy this message and any attachments.*

**Comment by lorengordon** [comment]
looks good, :ship:

**Comment by chriswvu** [comment]
I didn't push the change yet to add the new line character but looks good
in github.

Do I still need to push and create a pull request ?

I knew yaml was picky but didn't realize it needed that newline character
for last line.

Chris

On Fri, Feb 26, 2016 at 9:17 AM, Loren Gordon <notifications@github.com>
wrote:

> Merged #8 <https://github.com/plus3it/systemprep-content/pull/8>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/plus3it/systemprep-content/pull/8#event-567571136>.
>



-- 
Plus3 IT Systems - Cloud Engineer
304-279-4771
christopher.stotler@plus3it.com

-- 
 

------------------------------

*This e-mail and any attachments are intended only for the use of the 
addressee(s) named herein and may contain proprietary information. If you 
are not the intended recipient of this e-mail or believe that you received 
this email in error, please take immediate action to notify the sender of 
the apparent error by reply e-mail; permanently delete the e-mail and any 
attachments from your computer; and do not disseminate, distribute, use, or 
copy this message and any attachments.*

**Comment by lorengordon** [comment]
The change was made in commit #8c71510 in your own fork, so you must have pushed to your origin. When a PR is open, and you push a new commit, the PR is updated automatically to include the new commit.

So, all done, everything is good. :grin:

**Pull request merged**